G2P.pm

```diff
@@ -1342,7 +1342,8 @@ sub read_gene_data_from_file {
         my $confidence_category = $tmp{"DDD category"} || $tmp{"confidence category"}; # deprecate use of DDD category
```

Comment:
Running the plugin is returning the following warning:
`Use of uninitialized value $confidence_category in string eq at ...`

In the new format,`confidence_category` is called `confidence`.